src/read_write/node_writer.rs

```diff
@@ -213,6 +213,15 @@ impl WriteLE for Vec<Vector3<f64>> {
     }
 }
 
+impl WriteLE for Vec<Point3<f64>> {
```

Comment:
This is now needed because our positions are saved as `Point3` instead of `Vector3`.